<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabGame</class>
 <widget class="QWidget" name="TabGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="btn_snake">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string>Snake</string>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btn_pong">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string>Pong</string>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btn_pacman">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string>PacMan</string>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_preview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
